roles/openshift_manageiq/vars/main.yml

```diff
@@ -27,6 +27,7 @@ manage_iq_tasks:
     - policy add-role-to-user -n management-infra admin -z management-admin
     - policy add-role-to-user -n management-infra management-infra-admin -z management-admin
     - policy add-cluster-role-to-user cluster-reader system:serviceaccount:management-infra:management-admin
+    - policy add-cluster-role-to-user image-auditor system:serviceaccount:management-infra:management-admin
```

Comment:
This role will need to be conditionalized on `openshift.common.version_gte_3_2_or_1_2` since the role doesn't exist prior to that.